### DISCUSSION TITLE
It would be great to have a RAY_APP_NAME

### DISCUSSION THREAD

**Discussion by nmirceac** [spatie/ray#335] [Ideas]
That variable that will be shown in Ray - with a clear visual indicator [currently I'm prepending manually in some cases Laravel's config('app.name') for clarity

Maybe in the future even filter by APP - although that's not immediately required

It's great for a debugging scenario where you have an API Endpoint and and API Client doing requests to that endpoint - would be great to know which RAYs are coming from each source (without extra engineering - e.g. prefix, different colors etc)

![image](https://user-images.githubusercontent.com/475346/109773304-b9473900-7c07-11eb-85ae-af9dd35e471c.png)

**Comment by freekmurze** [comment]
I think @AlexVanderbist added this yesterday, we'll release it soon! 👍

**Reply by nmirceac** [reply]
Thanks - looking forward to testing it
Cheers

**Reply by AlexVanderbist** [reply]
@freekmurze I'm afraid I didn't. We will be showing the hostname of the server where the Ray originated from, not the app name or an arbitrary string. I'll pitch the idea again to the team :)

**Reply by nmirceac** [reply]
@AlexVanderbist  - although I can see the value in that, if different apps are on the same (local)host - it won't really help the user differentiating between the two (or more) sources
thanks for the update

**Reply by AlexVanderbist** [reply]
yeah, the hostname label was added in support of SSH server connections :)
